### DISCUSSION TITLE
Different filters for different clients😀

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#5637] [Q&A] [answered]
Hi,

I want to add different filters for different clients😀It is possible to make it?

Thank you.
Best regards!

**Comment by ainar-g** [comment, marked as answer]
Not currently, apart from [rules with the `client` modifier](https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#client).  See #435.

**Reply by ammnt** [reply]
Thanks!
